package.json

```diff
@@ -1,13 +1,9 @@
 {
   "name": "libp2p-noise",
   "version": "1.1.2",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
-  "module": "lib/index.js",
+  "main": "dist/src/index.js",
   "files": [
-    "dist",
-    "lib",
-    "src"
```

Comment:
I would leave src as we need to provide sourceMaps so we can have info where error occured in typescript file rather than builded js file